sdks/go/pkg/beam/runners/universal/runnerlib/stage.go

```diff
@@ -43,7 +43,7 @@ func Stage(ctx context.Context, id, endpoint, binary, st string) (retrievalToken
 		return "", nil
 	}
 
-  return StageViaLegacyApi(ctx, cc, binary, st)
```

Comment:
This cleanup is  separate from the rest of the change.

Reply:
Came bundled in with `go fmt` but I'll move it to a separate PR.